src/components/Desktop/pages/DesktopDate.tsx

```
@@ -253,13 +252,213 @@ function MenuDisplay({date} : MenuDisplayProps){
+                  </ul>
+                ) : (
+                  <div className="p-4 text-center text-gray-400">조식 메뉴가 없습니다.</div>
+                );
+              })()}
+            </div>
+          </div>
+          <div className="flex-1">
+            <div className="text-center p-3 border-b-1 border-orange-500 font-semibold">
+              제2 학생식당
+            </div>
+            <div className="">
```

Biome (2.1.2)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)